core/templates/dev/head/pages/exploration_player/PlayerTranscriptServiceSpec.js

```diff
@@ -132,11 +134,11 @@ describe('Player transcript service', function() {
     expect(pts.getLastStateName()).toBe('Second state');
 
     expect(pts.getNumSubmitsForLastCard()).toBe(0);
-    pts.addNewAnswer('first answer', null);
+    pts.addNewInput('first answer', null);
     expect(pts.getNumSubmitsForLastCard()).toBe(1);
-    pts.addNewFeedback('first feedback');
+    pts.addNewResponse('first feedback');
     expect(pts.getNumSubmitsForLastCard()).toBe(1);
-    pts.addNewAnswer('second answer', null);
+    pts.addNewInput('second answer', null);
     expect(pts.getNumSubmitsForLastCard()).toBe(2);
```

Comment:
Should check the case when hints are added as well to make sure the counter does not increment.

Shouldn't we change addNewInput() to always take a second boolean arg (not null)?